mllib/src/test/scala/org/apache/spark/ml/tree/impl/TreeTests.scala

```diff
@@ -43,7 +43,6 @@ private[ml] object TreeTests extends SparkFunSuite {
       categoricalFeatures: Map[Int, Int],
       numClasses: Int): DataFrame = {
     val spark = SparkSession.builder()
-      .master("local[2]")
```

Comment:
here we build `SparkSession` with an existing spark context, so setting the `master` is useless.